apps/academy-wep/components/CTAButton.tsx

```
@@ -0,0 +1,22 @@
+import React, { forwardRef } from "react"; // Importing React and forwardRef
+
+const CTAButton = forwardRef<
+  HTMLButtonElement,
+  React.ButtonHTMLAttributes<HTMLButtonElement>
+>(({ className, ...props }, ref) => {
+  return (
+    <div className="flex justify-center md:justify-start">
+      <button
+        ref={ref}
+        className={`bg-gradient-to-r from-green-600 to-gray-900 text-white font-semibold py-3 px-6 my-4 rounded-xl shadow-lg hover:from-green-500 hover:to-green-700 transition duration-300 ${className}`}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)